articles/using-fleet-and-okta-workflows-to-generate-a-daily-os-report.md

```diff
@@ -0,0 +1,140 @@
+
+To visualize this iteration, it looks like such: 
+
+The first run:
+
+MacOS Versions and Counts:
+
+13.3.0: 4
+
+The second run:
+
+MacOS Versions and Counts:
+
+13.3.0: 4
+
+13.2.1: 2
+
+The third run:
+
+MacOS Versions and Counts:
+
+13.3.0: 4
+
+13.2.1: 2
+
+13.2.0: 2
+
+So on and so forth through all items in the list.
+
+Once it reaches the end of the list, it returns the value back to the main flow in a single block of text.
+
+
+![Okta workflow Return Card](../website/assets/images/articles/using-fleet-and-okta-workflows-image2-500x210@2x.png "Okta workflow Return Card")
+
+
+Back in the main flow, we can start to put it all together and build the Slack messages we will post. We first want a date to add, in a human-readable format. Using ISO formats, we manipulate the date to produce the easy-to-read output like **Thursday, May 4, 2023.** The documentation elaborates further on different values you can use to create other formats.  
+
+
+![Okta workflow Date to Text card](../website/assets/images/articles/using-fleet-and-okta-workflows-image5-400x431@2x.png "Okta workflow Date to Text card")
+
+
+With all of the parts needed for the post, we use a final `Compose` card to string it all together!
+
+
+![Okta workflow Compose card composing Slack message.](../website/assets/images/articles/using-fleet-and-okta-workflows-image8-500x426@2x.png "Okta workflow Compose card composing Slack message.")
+
+
+Using one of my favorite cards, the `Construct` we create the key:value pair for our Slack message. So, for example, to post to the Slack API, we need a couple of values like, `channel` and `text`. And since we are using our friendly bot, we throw in the `username` as well. Check out the Slack docs [here](https://api.slack.com/methods/chat.postMessage) for the different keys you can use in the chat.postMessage API method, it's extensive!
+
+```{
+
+  "username": "Milo",
+
+  "channel": "C0467T97NLF",
+
+  "text": ":cat-wave: We're monitoring the upgrade to 13.3.1, here is the daily MacOS status for *Wednesday, May 3, 2023*:\n\nMacOS Versions & Count:\n*13.3.1*: 471\n*13.3.0*: 4\n*13.2.1*: 2\n*13.0.0*: 1\n*12.5.1*: 6\n\n:fleet: Find out more details &lt;https://kibbleme.devices.com/dashboard/mac|here>! :computer: :fleet:\n"
+
+}
+```
+
+Lastly, we use a `POST` card to send this to Slack. 
+
+
+![Okta workflow Post card to send message to Slack.](../website/assets/images/articles/using-fleet-and-okta-workflows-image3-352x1006@2x.png "Okta workflow Post card to send message to Slack.")
+
+
+That's really all there is to it! You could send this data almost anywhere, like if you prefer a Jira ticket or even a Google Spreadsheet. We have this running daily for now but will switch to weekly shortly. 
+
+Additionally, you could parse the return values and use the `If Error` card to handle anything other than a 200, but I have skipped that for now.
```

Comment:
@spokanemac can you add [the required `<meta>` tags](https://fleetdm.com/handbook/marketing/article-formatting-guide#meta-tags) to this article?

```suggestion
Additionally, you could parse the return values and use the `If Error` card to handle anything other than a 200, but I have skipped that for now.

<meta name="articleTitle" value="Using Fleet and Okta workflows to generate a daily OS report">
<meta name="authorFullName" value="TODO">
<meta name="authorGitHubUsername" value="TODO">
<meta name="category" value="guides">
<meta name="publishedOn" value="2023-05-05">
<meta name="articleImageUrl" value="../website/assets/images/articles/using-fleet-and-okta-workflows-to-generate-a-daily-os-report@2x.jpg">
<meta name="description" value="TODO">
```

Reply:
Added. Note: this is set to publish next Tuesday.